src/PrestaShopBundle/Resources/config/adapter/services.yml

```diff
@@ -4,6 +4,10 @@ services:
     _defaults:
         public: true
 
+    _instanceof:
+        PrestaShop\PrestaShop\Core\Form\AbstractFormHandler:
```

Comment:
for some reasons, I can't type hint on the related interface: if someone have an idea?